frontend/packages/noobaa-storage-plugin/locales/en/noobaa-storage-plugin.json

```diff
@@ -138,6 +138,11 @@
   "Service Name": "Service Name",
   "System Name": "System Name",
   "Version": "Version",
+  "Create Namespace Store": "Create Namespace Store",
```

Comment:
you dont need to update this file.
revert this please.

Reply:
done